ietf/utils/migrations/0004_pause_to_change_database_engines.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.28 on 2022-11-10 09:27
+
+from django.conf import settings
+from django.db import migrations
+
+def forward(apps, schema_editor):
+    assert(settings.DATABASES['default']['ENGINE'] == 'django.db.backends.postgresql_psycopg2', 'Please change to use postgres before continuing')
```

Comment:
```suggestion
    assert settings.DATABASES['default']['ENGINE'] == 'django.db.backends.postgresql_psycopg2', 'Please change to use postgres before continuing'
```
Brace yourself for a groan - the `assert` keyword cannot be used with parentheses around its arguments. It's not a method call, so the parentheses mean you're asserting the tuple `(False, 'Please change...')` which is truthy regardless of whether its first element is `True` or `False`.